examples/zola/vercel.json

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "ZOLA_VERSION": "0.19.1"
```

Comment:
In the modern (Node 20) build container, we don't preinstall any binaries like zola or hugo. Instead, it we need to specify the version to be used with an env var like this one.